TypeCobol.Test/TestCollection.cs

```diff
@@ -310,5 +310,42 @@ public void CheckDocumentation()
             Console.Write("Number of tests: " + nbOfTests + "\n");
             Assert.IsTrue(nbOfTests > 0, "No tests found");
         }
+
+#if EUROINFO_RULES
+        [TestMethod]
+        //[Ignore]
+        [TestCategory("Parsing")]
+        [TestProperty("Time", "fast")]
+        public void CheckToBeIndividuallyExecuted()
+        {
+            var errors = new System.Collections.Generic.List<Exception>();
+            int nbOfTests = 0;
+            string[] extensions = { ".cbl", ".pgm" };
+            string[] compilerExtensions = extensions.Concat(new[] { ".cpy" }).ToArray();
+
+            string directory = PlatformUtils.GetPathForProjectFile("Parser" + Path.DirectorySeparatorChar + "ToBeIndividuallyExecuted");
+            var folderTester = new FolderTester(directory, directory, directory, extensions, compilerExtensions);
+            try
+            {
+                folderTester.Test(false, false, false, true);
+            }
+            catch (Exception ex)
+            {
+                errors.Add(ex);
```

Comment:
I don't get the logic here. As soon as there is one exception, you'll leave the folderTester.Test method so there can be only one exception. The `errors` list is useless.
I think you'll need to iterate over the files to have true individual testing but there should be no need for that.